public/v2/apps/shopware.json

```diff
@@ -0,0 +1,117 @@
+                }
+            },
+            "$$cap_appname-shopware": {
+                "depends_on": [
+                    "$$cap_appname-db"
+                ],
+                "image": "shyim/shopware:$$cap_sw_version",
+                "restart": "always",
+                "environment": {
+                    "DATABASE_HOST": "srv-captain--$$cap_appname-db:3306",
+                    "DATABASE_URL": "mysql://$$cap_db_user:$$cap_db_pass@srv-captain--$$cap_appname-db:3306/shopware",
+                    "DB_USER": "$$cap_db_user",
+                    "DB_PASSWORD": "$$cap_db_pass",
+                    "APP_URL": "http://$$cap_appname-shopware.$$cap_root_domain",
+                    "APP_SECRET": "$$cap_app_scret",
+                    "INSTANCE_ID": "$$cap_instance_id",
+                    "MAILER_URL": "$$cap_mailer_url",
+                    "INSTALL_LOCALE": "$$cap_sw_locale",
+                    "INSTALL_CURRENCY": "$$cap_sw_currency",
+                    "INSTALL_ADMIN_USERNAME": "$$cap_sw_admin_username",
+                    "INSTALL_ADMIN_PASSWORD": "$$cap_sw_admin_password"
+                },
+                "volumes": [
+                    "$$cap_appname-state:/state",
+                    "$$cap_appname-custom-plugins:/var/www/html/custom/plugins",
+                    "$$cap_appname-files:/var/www/html/files",
+                    "$$cap_appname-logs:/var/www/html/var/logs",
+                    "$$cap_appname-theme:/var/www/html/public/theme",
+                    "$$cap_appname-media:/var/www/html/public/media",
+                    "$$cap_appname-bundles:/var/www/html/public/bundles",
+                    "$$cap_appname-sitemap:/var/www/html/public/sitemap",
+                    "$$cap_appname-thumbnail:/var/www/html/public/thumbnail",
+                    "$$cap_appname-jwt:/var/www/html/config/jwt"
+                ]
+            }
+        }
+    },
+    "instructions": {
+        "start": "Shopware is a trendsetting ecommerce platform to power your online business.",
+        "end": "Shopware is successfully deployed. Please wait a minute before accessing the shop."
+    },
+    "variables": [
+        {
+            "id": "$$cap_instance_id",
+            "label": "Instance ID (Can be generated with openssl rand -hex 32)",
+            "defaultValue": "10612e3916e153dd3447850e944a03fabe89440970295447a30a75b151bd844e"
+        },
+        {
+            "id": "$$cap_app_scret",
+            "label": "App Secret (Can be generated with openssl rand -hex 32)",
+            "defaultValue": "440dec3766de53010c5ccf6231c182acfc90bd25cff82e771245f736fd276518"
+        },
+        {
+            "id": "$$cap_db_user",
+            "label": "Database user",
+            "defaultValue": "shopwareuser",
+            "validRegex": "/^([a-zA-Z0-9])+$/"
+        },
+        {
+            "id": "$$cap_db_pass",
+            "label": "Database password",
+            "description": "",
+            "validRegex": "/.{1,}/"
```

Comment:
You probably want to limit this to alpha-numeric values - because otherwise your password in your database URL will not work properly.

Reply:
I did copycat that from wordpress (https://github.com/caprover/one-click-apps/blob/master/public/v2/apps/wordpress.json#L57)
Should I create another PR? 🙈